frontend/src/index.jade

```diff
@@ -145,8 +145,8 @@ html
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title
                 .summary-chart__title--index {{ i+1 }}
-                .summary-chart__title--repo(v-if="filterGroupSelection === 'groupByNone'") {{ user.repoPath }}
-                .summary-chart__title--author-repo(v-if="filterGroupSelection === 'groupByAuthors'") {{ user.repoPath }}
+                .summary-chart__title--repo(v-if="filterGroupSelection === 'groupByNone'") {{ user.repoName }}
+                .summary-chart__title--author-repo(v-if="filterGroupSelection === 'groupByAuthors'") {{ user.repoName }}
```

Comment:
We seem to have `repoName` and `repoPath` for the same value. If it is the case, remove one.

Reply:
Yes I did remove repoPath.

Reply:
What I mean is to remove all the occurrence of `repoPath` and this attribute together. We should not have duplicate arguments which may drag down our performance.

Reply:
I removed the `repoPath` attribute and kept only the `repoName` attribute. After that, I replaced all occurrence of `repoPath` to `repoName` since they both refer to the same value. It is done in 0b13169d74ab86be4d699299ddab742f01ee2080. Is this what you meant?

Reply:
Sorry, I didn't see the change, my bad.